.github/workflows/ci.yaml

```diff
@@ -18,15 +18,13 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Set up Python
-        uses: actions/setup-python@v5
+        uses: pdm-project/setup-pdm@v4
         with:
           python-version: "3.13"
-
-      - name: Install PDM
-        run: pipx install pdm
+          cache: true
 
       - name: Install dependencies
-        run: pdm sync && pdm install --dev
+        run: pdm install --dev
```

Comment:
`pdm install` includes _all_ groups, not just the explicit `all` group within dev deps, that's just a shortcut so that locally you can `pdm install -dG all`.